assets/src/edit-story/components/canvas/singleSelectionMoveable/index.js

```diff
@@ -71,6 +78,25 @@ function SingleSelectionMoveable({
     latestEvent.current = pushEvent;
   }, [pushEvent]);
 
+  // If scroll ever updates, update rect now
+  useEffect(() => {
```

Comment:
Maybe it would be good to create a hook for it later. This will also be needed for the Moveable used in the edit mode later when Zoom will work for edit mode, too.